src/wp-includes/html-api/class-wp-html-open-elements.php

```diff
@@ -144,6 +144,28 @@ public function current_node() {
 		return $current_node ? $current_node : null;
 	}
 
+	/**
+	 * Checks if the node at the top of the stack matches provided node name.
+	 *
+	 * @example
```

Comment:
I believe that this style of example only comes over from the Gutenberg end, and to play well with the WordPress documentation we should follow this pattern

```php
/**
 *
 * Example:
 *
 *     // Is the current node a text node?
 *     $stack->current_node_is( '#text' );
 *
 *     // Is the current node a DIV element?
 *     $stack->current_node_is( 'DIV' );
 */
```

That is: `Example:`, a blank line, and then four spaces to indent the code. This works well with various IDE support thankfully.